components/compliance-service/reporting/relaxting/util.go

```diff
@@ -73,6 +73,22 @@ func (backend *ES2Backend) ES2Client() (*elastic.Client, error) {
 	return esClient, err
 }
 
+func (backend *ES2Backend) ReindexStatus(ctx context.Context, taskID string) (bool, error) {
+	tasksGetTaskResponse, err := elastic.NewTasksGetTaskService(esClient).
+		TaskId(taskID).
+		WaitForCompletion(false).
+		Do(ctx)
+	if err != nil {
+		return false, err
+	}
+
+	if tasksGetTaskResponse.Task == nil {
+		return false, fmt.Errorf("ReindexStatus: task %s not found", taskID)
```

Comment:
If the Task from the `taskID` is not found in Elasticsearch, we are throwing an error. This could happen if Elasticsearch is restarted.

Reply:
We should not be throwing an error here. I have found for some Elasticsearch instances that the `tasksGetTaskResponse.Task` is `nil`, but this does not mean there is an error.